docs/insights-archive-sample/config/workload_info.json

```diff
@@ -22,7 +22,8 @@
         "sha256:1bd85e07834910cad1fda7967cfd86ada69377ba0baf875683e7739d8de6d1b0"
       ],
       "firstCommand": "icTsn2s_EIax",
-      "firstArg": "2v1NneeWoS_9"
+      "firstArg": "2v1NneeWoS_9",
+      "repository":"quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:a16c19e8f7372bec6e2b62f1eae6d4796d6aec8435679c7821fdc267fa667140"
```

Comment:
This must be hashed similarly as other values please.

Reply:
- [ ]  Detect where in the code the values of the other properties are being hashed. Then hash this value as well.
- [ ] Update sample file